docs/console/console-extensibility/sso.md

```diff
@@ -1,86 +1,164 @@
+       |    |                                         ^      v
+      (A)  (C)                                        |      |
+       |    |                                         |      |
+       ^    v                                         |      |
+     +---------+                                      |      |
+     |         |>---(D)-- Authorization Code ---------'      |
+     |  Client |          & Redirection URI                  |
+     |         |                                             |
+     |         |<---(E)----- Access Token -------------------'
+     +---------+       (w/ Optional Refresh Token)
+
+:::info
+When accessing an extension from within the Mia-Platform Console, the user is already logged in the system thus, when starting a new OAuth 2.0 flow for an extension, the user grant is considered implicit and no further action is expected from the user.
+
+This means that in the sequence above, the (B) part is done without requiring any user action. This behavior may however change in the future.
+:::
+
+## Extension Login Endpoint
+
+The *Login Endpoint* is the endpoint exposed by the extension frontend and is needed to start an authorization flow.
+
+When navigating to the *Login Endpoint*, the extension generates a new `state` that is used throughout all the login flow.
+
+It does not matter whether this endpoint shows some UI elements to inform the user about the login or not. What matters is that - in order to perform the login - this page MUST redirect the *user-agent* (the browser in use) to the specific extension authorization endpoint of the Console, specifying the generated `state`.
+
+The Console endpoint that handles extension authorization is the following one, preceded by the Console base URL:
+
+`/tenants/{tenantId}/extension/{extensionId}/authz?state={state}`
+
+Here an example:
+
+```js
+function performExtensionLogin() {
+   const tenantId = 'acme'
+   const extensionId = 'acme-extension-123'
    
-     ```json
-     {
-       "code": "CODE",
-       "state": "STATE"
-     }
-     ```
-
-5. **Receive Access Token**
-   - The Console will respond with an access token. The extension's backend returns this token to its frontend, where it can be used to access protected resources in the backend.
-
-6. **Validate the Access Token**
-   - The extension backend can validate the token by calling the following Console endpoint:
+   // Remember to store the state and validate it throughout the login flow
+   const state = 'foo'
    
-     ```
-     GET CONSOLE_BASE_URL/jwks
-     ```
+   window.location.replace(`https://console.cloud.mia-platform.eu/tenants/${tenantId}/extension/${extensionId}/authz?state=${state}`)
+}
+```
+
+```html
+<!-- When user clicks the button the login flow is started -->
+<button onclick="performExtensionLogin()">Login with Mia-Platform</button>
+```
+
+:::info
+How the extension generates and stores the `state` is not scope of this guide.
```

Comment:
can we share some hints on how to properly generate it? (e.g. use a UUID or something similar)

Reply:
added more detail